docs/source/bond_family_guide/sysadmin_guide/installation.rst

```diff
@@ -0,0 +1,477 @@
+
+  + The configuration steps to configure SGX may differ from those in this document
+  + The hardware must support `Intel SGX <https://software.intel.com/en-us/sgx>`_.
+
+- Supported OS versions: Windows 10 and Windows 7
+
+- If you need the network drivers for your HP workstation, they can downloaded
+  `here <http://ftp.hp.com/pub/softpaq/sp72501-73000/sp72518.exe>`_. 
+
+- Installation of the validator requires the use of a service account. The 
+  validator will not run under a user with admin privileges. 
+
+
+Networking
+==========
+
+Networking for your validator node must be set up before your node will be able
+to communicate with other nodes in the network:
+
+- Your computer or device must be connected to your LAN/WAN via the integrated 
+  network interface.
+- The node requires a publicly-addressable IP address or fully-qualified 
+  domain name (FQDN).
+- The node may be located behind a NAT firewall, may be given a fully exposed 
+  public network address, or may be on a DMZ.
+
+The following additional requirements apply:
+
+- An open TCP port on the publicly addressable IP address that maps to the 
+  node's configured TCP/http port.
+  
+  + The default TCP port is 8800, but your configuration may differ. 
+
+- An open UDP port on the publicly addressable IP address that maps to 
+  node's configured UDP/gossip layer. 
+  
+  + The default UDP port is 5500, but your configuration may differ. 
+
+Configuration of the validator node, including networking, is covered in
+:doc:`configuration`.
+
+For technical questions related to network, please see :doc:`contacts`. 
+
+
+Before You Begin
+================
+
+Download Required Software
+--------------------------
+
+A number of software packages and tools are required by Sawtooth Lake.
+Download the required software by clicking the links below.
+
+.. note:: The Sawtooth Lake validator software is provided to you by your 
+    Intel representative. Refer to :doc:`contacts`.
+
+1. `Python 2.7 for Windows <https://www.python.org/ftp/python/2.7.11/python-2.7.11.amd64.msi>`_
+
+  - Filename: ``python-2.7.11.amd64.msi``
+
+2. `Python Extensions for Windows <https://sourceforge.net/projects/pywin32/files/pywin32/Build%20219/pywin32-219.win-amd64-py2.7.exe/download>`_
+
+  - Filename: ``pywin32-219.win-amd64-py2.7.exe``
+
+3.  Intel® Software Guard Extensions Platform Software 
+
+  - Filename: ``Intel(R)_SGX_Windows_x64_PSW_1.6.101.32869.msi``
+  - Provided to you by your Intel representative
+
+4. `Intel® Management Engine (Intel® ME) Software <https://downloadmirror.intel.com/25881/eng/ME_Consumer_Win7_8.1_10_64_11.0.2.1002.zip>`_
+
+  - Filename: ``ME_Consumer_Win7_8.1_10_64_11.0.2.1002.zip``
+
+5. The Sawtooth Lake validator
+
+  - Filename: ``sawtooth-validator.zip``
+  - Provided by your Intel representative
+
+.. _`Enable SGX`:
+
+Step 1: Enable SGX
+==================
+
+By default, Intel Software Guard (SGX) is disabled on your provided HP device. 
```

Comment:
Removed. 
Noting that the entire section may need to be revised or greatly generalized for unknown hardware.